### PR TITLE
Added "debugging" section to vscode-readme

### PR DESCRIPTION
**Pull request by yuvalsw**
This change is [Reviewable](https://reviewable.io/reviews/starkware-libs/cairo/2578)

### CODE REVIEW

File: vscode-cairo/README.md [hunk 1]
```diff
@@ -48,3 +48,13 @@ If successful, go back to `sudo npm install -g vsce` and continue from there.
    ```
 3. Reload VSCode.
 4. Press F5.
+
+# Logging
+
+If you want the logs to be printed in your VSCode terminal when you click “Debug”/”Run” from the CodeLens above the test, add this to your ~/.config/Code/User/settings.json:
+```
+"rust-analyzer.runnableEnv": {
+   "RUST_LOG": "debug,salsa=off,minilp=off"
```
Comment (spapinistarkware) [inline vscode-cairo/README.md:57]: Is this the place to add the CARGO_MANIFEST thingy to make debug from vscode work? If so, maybe write it as well?

**Review by spapinistarkware — CHANGES_REQUESTED** [review]
See inline comments

**Review by spapinistarkware — COMMENTED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/cairo/2578)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @yuvalsw)

**Review by yuvalsw — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/cairo/2578)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @spapinistarkware)

___
*[`vscode-cairo/README.md` line 57 at r1](https://reviewable.io/reviews/starkware-libs/cairo/2578#-NR43kr4uYj-sBz6w5yH-r1-57:-NRSWv5L1SlFfl41Y6PQ:bsmd4l2) ([raw file](https://github.com/starkware-libs/cairo/blob/744e30eab66ba8e03fda8415192919c1037a96fe/vscode-cairo/README.md#L57)):*
<details><summary><i>Previously, spapinistarkware (Shahar Papini) wrote…</i></summary><blockquote>

Is this the place to add the CARGO_MANIFEST thingy to make debug from vscode work? If so, maybe write it as well?
</blockquote></details>

Yes. Done

**Review by spapinistarkware — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/cairo/2578#-:-NRSrqyZ6ctIvrqUYzZO:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @yuvalsw)

**Pull request merged**